roles/pgbackrest-backup/tasks/pgbackrestbackup.yml

```diff
@@ -1,15 +1,3 @@
-- name: Editing pgbackrest.conf file
```

Comment:
you want to remove this step from "pgbackrest-restore" to make it consistent ??

Reply:
The task is to edit "pgbackrest.conf" file and not "postgres.conf" file. This conf file is needed if pgbackrest was not previously setup or it was lost somehow.

Reply:
If you recommend that, I can add a, "do this only if the file doesn't exist" clause to the task.